pkg/container-collection/container-collection.go

```diff
@@ -311,6 +325,31 @@ func (cc *ContainerCollection) Enrich(event *eventtypes.CommonData, mountnsid ui
 	}
 }
 
+func (cc *ContainerCollection) EnrichByNetNs(event *eventtypes.CommonData, netnsid uint64) {
+	event.Node = cc.nodeName
+
+	containers := cc.LookupContainersByNetns(netnsid)
+	if len(containers) == 0 || containers[0].HostNetwork {
+		return
+	}
+	if len(containers) == 1 {
+		event.Container = containers[0].Name
+		event.Pod = containers[0].Podname
+		event.Namespace = containers[0].Namespace
+		return
+	}
+	if containers[0].Podname != "" && containers[0].Namespace != "" {
+		// Kubernetes containers within the same pod.
+		// TBD: Should we verify that all containers belong to the same pod?
```

Comment:
IMO, it is not necessary. WDYT?

Reply:
As we want local-gadget to be used for all containers outside of k8s too, i think we should not verify as you suggested